app/controllers/spree/paybright_controller.rb

```diff
@@ -63,13 +63,11 @@ def handle_callback_params!
       rescue StandardError => e
         logger.info "PAYBRIGHTDEBUG Order number: #{@payment.order.number}: Error: #{e.message}"
         logger.info "PAYBRIGHTDEBUG: Response code: #{@payment.response_code}"
+
         if @payment.response_code.present?
-          @payment.update_column(:state, :checkout) if @payment.invalid?
-          @payment.void
+          @payment.update(state: :completed) if !@payment.completed?
```

Comment:
Could we add a log when we're updating to complete when not completed, so we can go and look back at how often this is happening?